### PR TITLE
[pt-br] Tradução de metricas para Portugues Brasil

### PR DESCRIPTION
**Pull request by cassioliveiram**
@edsoncelio pode dar uma olhada?

### CODE REVIEW

File: content/pt/docs/concepts/signals/metrics.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: Métricas
+weight: 2
+description: Uma medição capturada em tempo de execução.
+default_lang_commit: 09f39d1aef5d792a68be42746a06ac828a499350
+---
+
+Uma **métrica** é uma **medição** de um serviço em tempo de execução. O momento
+de captura dessas medições é conhecido como **evento de métrica**, que consiste
+não apenas na medição em si, mas também no momento em que ela foi capturada e os
```
Comment (theletterf) [inline content/pt/docs/concepts/signals/metrics.md:10]: Not sure if the original had **bold**, but I would not use it here.

Comment (theletterf) [reply content/pt/docs/concepts/signals/metrics.md:10]:
```suggestion
Uma métrica é uma medição de um serviço em tempo de execução. O momento
de captura dessas medições é conhecido como evento de métrica, que consiste
não apenas na medição em si, mas também no momento em que ela foi capturada e os
```

File: content/pt/docs/concepts/signals/metrics.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+nosso código.
+
+## Meter Provider
+
+Um Meter Provider (às vezes chamado de `MeterProvider`) é uma fábrica para
+`Meter`s (Medidas). Na maioria dos aplicativos, um Meter Provider é inicializado
+uma vez e seu ciclo de vida corresponde ao ciclo de vida do aplicativo. A
+inicialização do Meter Provider também inclui a inicialização do Resource e do
+Exporter. Normalmente, é o primeiro passo na medição com OpenTelemetry. Em
+alguns SDKs, um Meter Provider global já está inicializado para você.
```
Comment (theletterf) [inline content/pt/docs/concepts/signals/metrics.md:31]: I'd also avoid capitalizing words, even if the original is doing it (we need to change that, IMHO). Unless it's code entities like `MeterProvider`. +CC @svrnm

**Comment by edsoncelio** [comment]
@cassioliveiram thank you so much for your contribution!!
Unfortunately, we must mark this as duplicated because this one (#5120) was opened first and is almost ready to be merged.

Could you close, please?

**Comment by cassioliveiram** [comment]
Thank you for the inputs @theletterf , seems like this one is duplicated from https://github.com/open-telemetry/opentelemetry.io/pull/5120, I'll refer you comments there just for guarantee.

**Pull request closed without merging**